packages/salesforcedx-sobjects-faux-generator/test/fauxgeneratortest.ts

```diff
@@ -26,17 +26,33 @@ describe('generate fields set', function() {
     expect(classText).to.include('String Foo;');
```

Comment:
Please take a look at the other tests classes and

1. See how the file is named.
1. See how the it(....) clauses are named.

Let's be consistent.

Reply:
will do - this one has been out there for a while, but I'll be glad to change it for consistency.